src/components/gravid/Actions.ts

```diff
@@ -18,7 +18,8 @@ export enum Actions {
   Validate,
   Progress,
   HandleResponse,
-  Kvittering
+  Kvittering,
+  CloseServerFeilModal
```

Comment:
Synes det er nok med ServerFeil her. Fire ord for å beskrive blir mindre lesbart.